Doc/library/types.rst

```diff
@@ -239,7 +239,7 @@ Standard names are defined for the following types:
       The :term:`loader` which loaded the module. Defaults to ``None``.
 
       This attribute is to match :attr:`importlib.machinery.ModuleSpec.loader`
```

Comment:
The errors in this file have existed since 3.7, so if you'd like, the changes can be backported to 3.9 and 3.10. The whatsnew in 3.6 file can also be backported to 3.9 and 3.10.

Do tell me if you need help with backports, I'll be happy to help!

Reply:
I don't feel this need to be backported, I expect people to read the changelogs from /3/ anyway, I'd say "don't spend too much time with it".

Reply:
Alright